celery/canvas.py

```diff
@@ -1487,14 +1487,16 @@ def __length_hint__(self):
 
     def run(self, header, body, partial_args, app=None, interval=None,
             countdown=1, max_retries=None, eager=False,
-            task_id=None, **options):
+            task_id=None, partial_kwargs=None, **options):
```

Comment:
what is the purpose of partial_kwqrs?

Reply:
The short answer is I think it will be ok to rename it to `kwargs`. I initially called it`partial_kwargs` to keep to the same naming converting with `partial_args` as I didn't fully grasp why `partial_args` was called `partial_args` and not just `args`, but I think I understand it now. It makes sense that `partial_args` is called partial because the results from the tasks will be the last arg.

If I missed your point and the real question is why do we even need the kwargs argument, it's because this is the async equivalent of [this](https://github.com/celery/celery/blob/59263b0409e3f02dc16ca8a3bd1e42b5a3eba36d/celery/canvas.py#L1435): when we eagerly apply the chord, the `apply` gets  both the `args` and `kwargs` while the true async call does not receive the kwargs. It is needed to fix the omission in the api. 

At the moment, I just pushed an updated commit that addresses the tests failing with the redis backend. If that's ok, then I'll update this PR with renaming `partial_kwargs` to `kwargs`